test/Tests/src/Data/Time/Date_Time_Spec.enso

```diff
@@ -45,7 +44,7 @@ spec =
             (Date_Time.new 2022 12 12).should_not_equal (Date_Time.new 1996)
 
 default_zone_equal z1 z2 =
-    z1 . should_equal z2
+    z1 . should_equal frames_to_skip=1 z2
     False
 
 spec_with name create_new_datetime parse_datetime nanoseconds_loss_in_precision=False loose_zone_equal=default_zone_equal =
```

Comment:
Can we remove the `loose_zone_equal` altogether and just rely on `should_equal` directly now since this is fixed?

Reply:
I'd like to, but f2b22405d505c07733a2a52b7a5cfc6ea460ad54 seems to be the best I can get. No [advice on stack overflow](https://stackoverflow.com/questions/19654578/python-utc-datetime-objects-iso-format-doesnt-include-z-zulu-or-zero-offset) either.